frontend/storm-surge-hazards/style.css

```
@@ -0,0 +1,632 @@
+    --accent-critical: #ef4444;
+    --accent-success: #10b981;
+
+    --gradient-hero: linear-gradient(135deg, #0b1325 0%, #0c2340 50%, #0f172a 100%);
+    --gradient-surge: linear-gradient(90deg, #0284c7 0%, #06b6d4 100%);
+
+    --shadow-subtle: 0 4px 20px rgba(0, 0, 0, 0.25);
+    --shadow-glow: 0 0 25px rgba(56, 189, 248, 0.2);
+    --radius-md: 12px;
+    --radius-lg: 16px;
+
+    --font-heading: 'Outfit', -apple-system, BlinkMacSystemFont, sans-serif;
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)